roadmap.md

```diff
@@ -0,0 +1,249 @@
+    <td>December 2022</td>
+  </tr>
+  <tr>
+    <td>Point-in-time recovery (PITR)</td>
+    <td>Table-level and database-level PITR</td>
+    <td>BR supports table-level or database-level PITR.</td>
+    <td>October 2022</td>
+  </tr>
+  <tr>
+    <td rowspan="2">Data replication to downstream systems via TiCDC</td>
+    <td>Reduce TiCDC replication latency in planned offline scenarios</td>
+    <td>When TiKV, TiDB, PD, or TiCDC nodes are offline in a planned maintenance window, the replication latency of TiCDC can be reduced to less than 10 seconds.</td>
+    <td>October 2022</td>
+  </tr>
+  <tr>
+    <td>Support replicating data to object storage such as S3.</td>
+    <td>TiCDC supports replicating data changes to common object storage services.</td>
+    <td>November 2022</td>
+  </tr>
+  <tr>
+    <td>Data migration</td>
+    <td>TiDB Lightning supports table-level and partition-level incremental data import.</td>
+    <td>TiDB Lightning provides comprehensive table-level and partition-level data import capabilities</td>
+    <td>December 2022</td>
+  </tr>
+</tbody>
+</table>
+
+## Security
+
+<table>
+<thead>
+  <tr>
+    <th>Scenario</th>
+    <th>Feature</th>
+    <th>Description</th>
+    <th>Estimated Time of Delivery</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">ShangMi (SM) algorithms</td>
+    <td>Encryption-at-rest (TiKV and TiFlash) supports the SM4 algorithm.</td>
+    <td>Supports encrypting data stored in TiKV and TiFlash based on the SM4 algorithm.</td>
+    <td>September 2022</td>
+  </tr>
+  <tr>
+    <td>TiDB authentication supports the SM3 algorithm. </td>
+    <td>Provide a user authentication plugin based on the SM3 algorithm, which encrypts the password using the SM3 algorithm.</td>
+    <td>September 2022</td>
+  </tr>
+  <tr>
+    <td>Log redaction</td>
+    <td><ul><li>Data redaction in execution plans in TiDB Dashboard</li><li>Data redaction in TiDB-related logs</li></ul></td>
+    <td>Redact sensitive information in execution plans and various logs to enhance the security of user data.</td>
+    <td>October 2022</td>
+  </tr>
+  <tr>
+    <td>Password complexity check</td>
+    <td>A strong password is required.</td>
+    <td>To improve security, empty passwords and weak passwords are not allowed.<br>The required password length is not less than 8. The password must contain an uppercase letter, a lowercase letter, a number, and a character.</td>
+    <td>November 2022</td>
+  </tr>
+  <tr>
+    <td>Password expiration</td>
+    <td>TiDB provides password expiration management and requires users to change passwords regularly.</td>
+    <td>Reduce the security risk of password cracking or leakage caused by using the same password for a long time.</td>
+    <td>December 2022</td>
+  </tr>
+  <tr>
+    <td>Password policy management</td>
+    <td>TiDB provides a password reuse mechanism and brute-force cracking prevention capabilities</td>
```

Comment:
Added a period for complete sentences to keep the style consistent.
```suggestion
    <td>TiDB provides a password reuse mechanism and brute-force cracking prevention capabilities.</td>
```

Reply:
done in [749ad0e](https://github.com/pingcap/tidb/pull/38002/commits/749ad0e640d7eadd43b00f857886f46bb14c5914)